src/test/Calculator.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from '../components/Calculator';
+
+describe('Calculator component unit test', () => {
+  test('Calculator Component Renders Properly', () => {
+    const tree = renderer.create(<Calculator />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Good job for adding some of the tests required for the project such as all the functionalities & scenarios of the `operate.js` and `calculate.js` modules should be covered using Jest, usage of Jest snapshots for testing all the components. However I noticed that you are missing another requirement for this project.
You should ensure that you also use `React Testing Library to simulate user interaction of your application`, Right now , the feature **of simulating user interaction** is not used in any component such as in the `Calculator.jsx`, `Home.jsx`, `Qusotes.jsx`,... make sure you add it. To help you how to do so, I suggest you refer **[this article for documentation](https://testing-library.com/docs/react-testing-library/example-intro)**

- [ ] As required for the project, kindly ensure that all your tests are passing as there is one test failing for your `App.js` component.
![Screenshot from 2023-07-13 21-33-25](https://github.com/PabloBona/Math-Magicians/assets/83042742/8567ce24-4cae-4ecb-886f-a650d438c7ab)

Reply:
I tried the App.test.js again and it runs on my PC with Windows and on my partner's PC with Windows too. 
Maybe you use Unix Os and the config problem is not mine.
If missing use React Testing Library to simulate user interaction.